esmvalcore/_data_finder.py

```diff
@@ -231,63 +245,60 @@ def _find_input_dirs(variable, rootpath, drs):
 
 def _get_filenames_glob(variable, drs):
     """Return patterns that can be used to look for input files."""
-    path_template = _select_drs('input_file', drs, variable['project'])
+    path_template = _select_drs("input_file", drs, variable["project"])
     filenames_glob = _replace_tags(path_template, variable)
     return filenames_glob
 
 
 def _find_input_files(variable, rootpath, drs):
-    short_name = variable['short_name']
-    variable['short_name'] = variable['original_short_name']
+    short_name = variable["short_name"]
+    variable["short_name"] = variable["original_short_name"]
     input_dirs = _find_input_dirs(variable, rootpath, drs)
     filenames_glob = _get_filenames_glob(variable, drs)
     files = find_files(input_dirs, filenames_glob)
-    variable['short_name'] = short_name
+    variable["short_name"] = short_name
     return (files, input_dirs, filenames_glob)
 
 
 def get_input_filelist(variable, rootpath, drs):
     """Return the full path to input files."""
-    # change ensemble to fixed r0i0p0 for fx variables
-    # this is needed and is not a duplicate effort
-    if variable['project'] == 'CMIP5' and variable['frequency'] == 'fx':
-        variable['ensemble'] = 'r0i0p0'
```

Comment:
I really do not like this. Changes like this are not backwards compatible, i.e. we have to change every recipe that uses `fx` variables for CMIP5 data.

Reply:
I guess we can generalize without the conditional on `/fx` which I totally overlooked and it's indeed extremely restrictive, but we can't do a `.*` coz that'll take forever and a half, maybe just a `*` on ensembles? Even that will take quite a bit with models with 30-40 ensembles in CMIP6